wrappers/services.go

```diff
@@ -59,25 +59,55 @@ func generateSnapServiceFile(app *snap.AppInfo) (string, error) {
 }
 
 // StartSnapServices starts service units for the applications from the snap which are services.
-func StartSnapServices(s *snap.Info, inter interacter) error {
+func StartSnapServices(s *snap.Info, inter interacter) (err error) {
+	hasReloaded := false
+	sysd := systemd.New(dirs.GlobalRootDir, inter)
+	var enabled []*snap.AppInfo
+	var started []*snap.AppInfo
+
+	defer func() {
```

Comment:
I did this and put it in a gist, because it's not clear to me that it's better: is this what you meant? https://gist.github.com/chipaca/d674ab043e053a01e947d8aa1acc2242
If it is, I'll look for a 3rd party's opinion :-)